exonum-light-client/README.md

```diff
@@ -15,16 +27,44 @@ If you are using Gradle, add this to your dependencies
 compile 'com.exonum.client:exonum-light-client:0.1-SNAPSHOT'
 ```
 
-## Getting Started
+## Examples
```

Comment:
do we need to put here examples of using commons? Such as generation keys, tx message creation etc

Reply:
I think no, but the example below misses a TransactionMessage instantiation, which I consider as useful.